start.sh

```diff
@@ -21,13 +21,21 @@ echo Running python startups
 python manage.py migrate
 
 if [ -z "${IS_CRON_POD}" ]; then
-    # Start Gunicorn processes
-    echo Starting Gunicorn.
+    if [ -z "{PTVSD_DEBUG}" ]; then
```

Comment:
do I still need to set the `PTVSD` to True in order to work locally even though I won't be using the remote debugging?

Reply:
I think if you want to debug at all (with VSCode) you'd have to set this to true.

Reply:
If you want to switch to VSCode I can show you how it works. :) I feel like PyCharm is what Jetbrains expects you to use with python. IntelliJ is more for Java but works with other languages.

Reply:
yes, if you enable the python plugin with IntelliJ I have the full version of PyCham.  when I launch the IDE it always says "IntelliJ" So that's where I got stuck in my head

Reply:
I see, it sounds like PyCharm might support ptvsd. If it doesn't it's probably something really similar.